packages/eslint-plugin/test/__file_snapshots__/types/no-const-enum/index.d.ts.lint

```diff
@@ -0,0 +1,5 @@
+No errors
+
+==== types/no-const-enum/index.d.ts ====
+
+    enum F {}
```

Comment:
I snapshotted even the "good" files; the runner doesn't do a great job of removing unused snapshots (at least in this special "file snapshot" addon).

Reply:
Unused baselines are now caught via a test, but I think leaving these to make sure we encode the expected state is probably valuable.